code-challenges.js

```diff
@@ -4,38 +4,143 @@
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+var fib = [0, 1];
 
+// i want to loop throught the index in the array
+       for (var i=fib.length; i < 11; i++) {
 
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+// i put the formula in
+        fib[i] = fib[i-2] + fib[i-1];
+    }
```

Comment:
good job! this logic totally works! now we just need to wrap it all up in a pretty little function :)